README.md

```diff
@@ -60,15 +60,13 @@ All the users opertaions happened while the container is running. If you used a
 If say, you want to create a user named  `tommy`, type the following command
 
 ```
-docker exec -ti openconnect ocpasswd -c /etc/ocserv/ocpasswd -g "Route,All" tommy
+docker exec -ti openconnect ocpasswd -c /etc/ocserv/ocpasswd tommy
```

Comment:
it helps the users to know about ``-g`` option

Reply:
`tommylau/ocserv` is targeted for users in China, and the default groups are for that, there are not defaults in my `ocserv` docker image.